server/protocols/webadmin/README.adoc

```diff
@@ -121,3 +121,85 @@ The answer looks like :
 Response codes :
  - 200 : The user name list was successfully retrieved
  - 500 : Internal error while retrieving the users
+
+== Administrating user mailboxes
+
+=== Creating a mailbox
+
+.bash
+====
+curl -XPUT http://ip:port/users/usernameToBeUsed/mailboxes/mailboxNameToBeCreated
+====
+
+Resource name usernameToBeUsed should be an existing user
+Resource name mailboxNameToBeCreated should not be empty
+
+Response codes :
+ - 204 : The mailbox now exists on the server
+ - 404 : The user name does not exist
+ - 500 : Internal error
+
+=== Creating a mailbox
+
+.bash
+====
+curl -XDELETE http://ip:port/users/usernameToBeUsed/mailboxes/mailboxNameToBeCreated
+====
+
+Resource name usernameToBeUsed should be an existing user
+Resource name mailboxNameToBeCreated should not be empty
+
+Response codes :
+ - 204 : The mailbox now does not exist on the server
+ - 404 : The user name does not exist
+ - 500 : Internal error
+
+=== Testing existence of a mailbox
+
+.bash
+====
+curl -XGET http://ip:port/users/usernameToBeUsed/mailboxes/mailboxNameToBeCreated
+====
+
+Resource name usernameToBeUsed should be an existing user
+Resource name mailboxNameToBeCreated should not be empty
+
+Response codes :
+ - 204 : The mailbox exists
```

Comment:
204 No Content

The server has fulfilled the request but does not need to return an entity-body.

Whe have no content here, hence 204